packages/website/src/content/06-03-multimodal-api-reference.md

```diff
@@ -0,0 +1,230 @@
+
```

Comment:
this file is generated from `npm run publish-docs`. I also removed a single superfluous doc line